about/index.html

```diff
@@ -67,7 +67,7 @@
       <!-- End Content -->
         <!-- Begin Footer -->
         <div class="footer">
-          <p>&copy; 2011-2016 Hack the Future</p>
```

Comment:
I just tried `&mdash;` in our HTML and it doesn't look as good as in your example:

![image](https://cloud.githubusercontent.com/assets/1616866/22663815/f590be2e-ec62-11e6-9a62-5415cac9b2ba.png)

Like it's too large or something with the fonts we're using? Open to suggestions.